janus/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for janus.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+deployment:
+  replicaCount: 2
+  minAvailable: 1
+  databaseDSN: "mongodb://janus-database:27017/janus"
+
+imagePullSecrets: []
+
+image:
+  repository: quay.io/hellofresh/janus
+  tag: latest
+  pullPolicy: IfNotPresent
+
+service:
+  name: ops-gateway
+  type: ClusterIP
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 8080
+    name: http
+  - protocol: TCP
+    port: 443
+    targetPort: 8080
+    name: https
+  - protocol: TCP
+    port: 8081
+    targetPort: 8081
+    name: http-private
+
+ingress:
+  enabled: false
+  name: ops-gateway
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
ingress class is handled by the open policy agent already, so this can be omitted.